api/location/models.py

```diff
@@ -3,15 +3,28 @@
 
 from helpers.database import Base
 
+
 from utilities.utility import Utility
+import enum
+
+
+class Country_type(enum.Enum):
+    Uganda = "Uganda"
+    Kenya = "Kenya"
+    Nigeria = "Nigeria"
+
+
+class TimeZone_type(enum.Enum):
```

Comment:
Always do not name classes using undersocores, use CapWords  instead refer https://www.python.org/dev/peps/pep-0008/#class-names